.vscode/launch.json

```diff
@@ -1,6 +1,12 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
What that does?

Reply:
Just a way for me to debug the tests by pressing f5. Can remove that, if not needed.

Reply:
Looks nice, keep it!